jquery.d.ts is missing event.currentTarget

- [x] I tried using the latest `jquery/jquery.d.ts` file in this repo and had problems.
- [x] I tried using the latest stable version of tsc. https://www.npmjs.com/package/typescript
- [x] I have a question that is inappropriate for [StackOverflow](https://stackoverflow.com/).  (Please ask any appropriate questions there).
- [x] I want to talk about `jquery/jquery.d.ts`.

---

There is no definition for event.currentTarget. This means that TypeScript thinks it's an EventTarget, when it should be an element.

See jQuery API: https://api.jquery.com/event.currentTarget/

---

My personal use case:
```
event.currentTarget.classList.toggle('active');
```
is rejected with `error TS2339: Property 'classList' does not exist on type 'EventTarget'.`

--- a/jquery/jquery-tests.ts
+++ b/jquery/jquery-tests.ts
@@ -1525,6 +1525,9 @@ function test_eventParams() {
     $(window).on('mousewheel', (e) => {
         var delta = (<WheelEvent>e.originalEvent).deltaY;
     });
+    $( "p" ).click(function( event ) {
+      alert( event.currentTarget === this ); // true
+    });
 }
 
 function test_extend() {

--- a/jquery/jquery.d.ts
+++ b/jquery/jquery.d.ts
@@ -442,6 +442,7 @@ interface JQueryDeferred<T> extends JQueryGenericPromise<T> {
  * Interface of the JQuery extension of the W3C event object
  */
 interface BaseJQueryEventObject extends Event {
+    currentTarget: Element;
     data: any;
     delegateTarget: Element;
     isDefaultPrevented(): boolean;

